src/sm/sm_main/debug/full-tracer.js

```diff
@@ -169,7 +169,7 @@ class FullTracer {
         //Fill context object
         const context = {};
         context.to = `0x${getVarFromCtx(ctx, false, "txDestAddr")}`;
-        context.type = (context.to === "0x0") ? "CREATE" : "CALL";
+        context.type = Number(getVarFromCtx(ctx, false, "isCreateContract")) ? "CREATE" : "CALL";
         context.to = (context.to === "0x0") ? "0x" : ethers.utils.hexlify(getVarFromCtx(ctx, false, "txDestAddr"));
```

Comment:
I would set `context.to` depending on `context.type` since it makes more sense. Therefore, line 171 could be deleted.
Also, `hexlify` does not do any padding. Since `RLP` will fail in case `to` does not have 20 bytes (or empty for deployment) I would pad 20 bytes always.
Change other `context.to` compatison with `0` and replace them by `context.type`